src/r_util.cpp

```diff
@@ -504,6 +506,37 @@ namespace rhost {
             return R_FalseValue;
         }
 
+        extern "C" SEXP save_project(SEXP id, SEXP project_name, SEXP dest_dir, SEXP temp_dir) {
```

Comment:
This is intended for files under the project folder (under RTVS). So, 'yes' for any set of files as long as they are expected to be under the project folder.

---
In reply to: [74644583](https://github.com/Microsoft/R-Host/pull/78#discussion_r74644583) [](ancestors = 74644583)